src/components/subreddit-form/subredditForm.style.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components/macro';
```

Comment:
I'd try to stick to one naming convention for files. This file uses `camelCase`, the tests and App use `PascalCase`, the pages use no casing (e.g. `searchpage`). For non-component files it's ok to switch (e.g to `camelCase` instead of `PascalCase`). But component naming should be consistent 😄

Reply:
I think I messed up the workflow 😅. Will fix all issues in the next PR.